pallets/did/src/origin.rs

```diff
@@ -69,6 +70,22 @@ impl<DidIdentifier: Clone, AccountId: Clone> kilt_support::traits::CallSources<A
 	}
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+impl<OuterOrigin, AccountId, DidIdentifier>
+	kilt_support::traits::GenerateBenchmarkOrigin<OuterOrigin, AccountId, DidIdentifier>
```

Comment:
b70a73ba497524591b5b90628797dfc7c2a47baf

Reply:
I would not like to do that because the `use` would need to be guarded by a ` #[cfg(feature = "runtime-benchmarks")]`. This way we don't have a `use`.